lib/css-assembler.js

```diff
@@ -76,10 +83,19 @@ function assemble(cssFiles, absolutePath, type, callback) {
             Fs.readFile(file, {encoding: 'utf-8'}, function(err, content) {
                 var matches = [],
                     match,
-                    importCssFileAlias;
+                    importCssFileAlias,
+                    cssFile;
 
                 if (! err) {
-                    ret[cssFileAlias + ext] = content;
+
+                    //Ensure all import paths are Unix paths for prod compat.
+                    if (options.bundle) {
+                        cssFile = Upath.toUnix(cssFileAlias + ext);
+                    } else {
+                        cssFile = cssFileAlias + ext;
```

Comment:
The options get added in the bundling phase, but this is also called from the `theme-assests` plugin https://github.com/bigcommerce/stencil-cli/blob/master/server/plugins/theme-assets/theme-assets.module.js#L61. When you are running stencil-cli on a Windows machine you don't want the paths converted to Unix because then compilation will fail because the file paths will be wrong and will lead to the issue that this ticket is fixing.